src/benchmark.cc

```diff
@@ -53,700 +83,272 @@ DEFINE_double(benchmark_min_time, 0.5,
               "of the benchmark execution, regardless of number of "
               "threads.");
 
-DEFINE_bool(benchmark_memory_usage, false,
-            "Report memory usage for all benchmarks");
-
 DEFINE_int32(benchmark_repetitions, 1,
              "The number of runs of each benchmark. If greater than 1, the "
              "mean and standard deviation of the runs will be reported.");
 
-DEFINE_int32(v, 0, "The level of verbose logging to output");
-DEFINE_bool(color_print, true, "Enables colorized logging.");
+DEFINE_bool(benchmark_use_picoseconds, false,
```

Comment:
should this be just wrapped in to how we report times? ie, report times with units.